internal/validator/cosignvalidator/cosign_validator.go

```diff
@@ -35,10 +34,16 @@ type CosignValidator struct {
 	Name string `validate:"required"`
 	// type of the validator (will always be "cosign")
 	Type string `validate:"eq=cosign"`
-	// url of the Rekor instance
-	Rekor string `validate:"url"`
+	// Rekor instance
+	Rekor *client.Rekor
+	// public key of the Rekor instance
+	RekorPubkey *cosign.TrustedTransparencyLogPubKeys
+	// certificate of the fulcio instance
+	FulcioCert *x509.CertPool
```

Comment:
i think i will keep it the way it is, so that connaisseur will always pull "fresh" certificates, should they change while connaisseur is installed somewhere. then there is no need to reinstall.